core/src/main/java/com/netease/arctic/op/ArcticHadoopTableOperations.java

```diff
@@ -42,7 +43,16 @@ public TableMetadata refresh() {
   @Override
   public void commit(TableMetadata base, TableMetadata metadata) {
     arcticFileIO.doAs(() -> {
-      super.commit(base, metadata);
+      try {
+        super.commit(base, metadata);
+      } catch (RuntimeException e) {
```

Comment:
we can distinguish the CommitFailedException in rename to avoid ignoring retry when commit conflict.

Reply:
SGTM.